flash/core/classification.py

```diff
@@ -153,7 +153,8 @@ def __init__(self, labels: Optional[List[str]] = None, multi_label: bool = False
             self.set_state(LabelsState(labels))
 
     def serialize(self, sample: Any) -> Union[int, List[int], str, List[str]]:
-        sample = sample[DefaultDataKeys.PREDS] if isinstance(sample, Dict) else sample
+        if isinstance(sample, Dict) and DefaultDataKeys.PREDS in sample:
```

Comment:
Does it have to be a Dict or is every kind of Mapping okay?

Reply:
Nope, cleaned that bit up :smiley: